src/cli/commands/crossmsg/send_cross.rs

```diff
@@ -47,15 +46,13 @@ impl CommandLineHandler for WhitelistPropagator {
 
 #[derive(Debug, Args)]
 #[command(about = "Whitelist propagators in the gateway actor")]
-pub(crate) struct WhitelistPropagatorArgs {
+pub(crate) struct SendCrossMsgsArgs {
     #[arg(long, short, help = "The JSON RPC server url for ipc agent")]
     pub ipc_agent_url: Option<String>,
     #[arg(long, short, help = "The address that owns the message in the subnet")]
     pub from: Option<String>,
     #[arg(long, short, help = "The subnet to whitelist")]
     pub subnet: String,
-    #[arg(help = "The message cid to whitelist")]
-    pub postbox_msg_cid: Cid,
-    #[arg(help = "The addresses to whitelist")]
-    pub to_add: Vec<String>,
+    #[arg(help = "The cross network message to send")]
```

Comment:
This is wrong, in `SendCross`  the argument is a `Message` that one wants to send to a subnet, not a string. As discussed, let's maybe remove this code for now, or add a comment with a huge `FIXME` at the beginning of the file, but not expose this to users.